src/test/fuzz.cpp

```diff
@@ -157,6 +169,16 @@ fuzz(std::string const& filename, el::Level logLevel,
     {
         clock.crank(true);
     }
+
```

Comment:
mmm, the main loop `tryRead` is a bit weird: I would expect to see not just `clock.postToCurrentCrank` but also a call to `crank` to consume the message: without this, messages are queued somewhere and there is no guarantee that they will be executed in the same order... 
I think a better option to deal with this problem (and shutdown), would be to use `Simulation` (with 2 nodes), and call in this loop something like `crankForAtMost(0.5 seconds, false)`

Reply:
Crank now after posting `sendMessage` to the crank. Thus messages actually send.